### PR TITLE
Add admin UI documentation, fix title for no-subdomain doc

### PR DESCRIPTION
**Pull request by paskal**
This PR:
- Add Admin UI documentation
- Fix title for no-subdomain doc
- Remove unused "System requirements" page of the doc
- Fix Readme links to documentation
- Compress Admin UI screenshots for smaller size

Feel free to nuke the wiki pages and remove them from the repo after that PR is merged, as all of them would be present in the new documentation.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ Remark42 is a self-hosted, lightweight and simple (yet functional) comment engin
 * Clean, lightweight and customizable UI with white and dark themes
 * Multi-site mode from a single instance
 * Integration with automatic SSL (direct and via [nginx-le](https://github.com/nginx-le/nginx-le))
-* [Privacy focused](#privacy)
+* [Privacy focused](https://remark42.com/#privacy)
```
Comment (akellbl4) [inline README.md:26]: I'm not sure if it's right link

Comment (umputun) [reply README.md:26]: yeah, this is the one https://remark42.com/privacy/

Comment (paskal) [reply README.md:26]: It led to a particular part of the text I'm referencing before the change: it was moved to the site's main page from the Readme but was not appropriately referenced before.

Comment (paskal) [reply README.md:26]: Commit: https://github.com/umputun/remark42/commit/19a4eada25fce21e73ac2bf310361c352f4f0683

**Pull request merged**